src/rust/engine/process_execution/src/local.rs

```diff
@@ -384,6 +402,21 @@ mod tests {
     )
   }
 
+  #[test]
+  fn expect_timeout() {
+    match run_command_locally(ExecuteProcessRequest {
+      argv: owned_string_vec(&["/usr/bin/yes"]),
```

Comment:
I imagine this will want to end up being something like:


`bash -c "echo foo ; echo bar >&2 ; touch some-file ; sleep 100 ; touch some-other-file"` expecting output files `some-file` and `some-other-file`
and assert that stdout and stderr are properly populated, that the exit code is -15, and that the output directory contains `some-file` but not `some-other-file`

Reply:
It would be good to confirm that the child process is actually TERM'd as well (or open a followup for that, since it might be extra work?)